### PR TITLE
Do not quantize logits

### PR DESCRIPTION
**Pull request by gobbleturk**
Quantizing logits hurts training accuracy

### CODE REVIEW

**Review by rwitten — APPROVED** [review]
Just to be clear the comment should be "we do not quantize the logit matmul"
